### ISSUE TITLE
Cocoapods and error with OS X 10.10

### ISSUE THREAD

**Issue by malcommac** [CocoaPods/CocoaPods#3140]
Hi,
I've just tried to push my cocoapods library using:
pod trunk push MyLib.podspec 

The result is this, any idea? I've also tried updating ruby to 2.2.0 and re-installing cocoapods but nothing.

```
Validating podspec
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:422:in `open': No such file or directory - /Users/daniele/.cocoapods/repos (Errno::ENOENT)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:422:in `foreach'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:422:in `children'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/lib/cocoapods/sources_manager.rb:63:in `all'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/lib/cocoapods/user_interface/error_report.rb:130:in `repo_information'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/lib/cocoapods/user_interface/error_report.rb:34:in `report'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/lib/cocoapods/command.rb:58:in `report_error'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/claide-0.7.0/lib/claide/command.rb:300:in `handle_exception'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/claide-0.7.0/lib/claide/command.rb:274:in `rescue in run'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/claide-0.7.0/lib/claide/command.rb:264:in `run'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/lib/cocoapods/command.rb:45:in `run'
	from /Users/daniele/.rvm/gems/ruby-2.2.0@global/gems/cocoapods-0.35.0/bin/pod:43:in `<top (required)>'
	from /usr/bin/pod:23:in `load'
	from /usr/bin/pod:23:in `<main>'
```

**Comment by orta** [comment]
This may be a 0.35 but around not creating a default repo, try run `pod setup` then re-running please

**Comment by segiddins** [comment]
Yup, what @orta said. This bug has been fixed in 0.36.
